### PR TITLE
add game started state

### PR DESCRIPTION
**Pull request by frgiovanna**
## Motivation

This PR adds the game started state on your-turn page. It also adjusts the typography to have a bigger one.

https://github.com/user-attachments/assets/e066b576-45ff-49b6-8826-852ea8b71dcb

### CODE REVIEW

File: src/shared/components/Typography/Typography.tsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
+import { Body1, Body2, Display, H1, H2, H3 } from './styles';
 import { TypographyProps } from './types';
 
 const typographyVariants = {
   Body1: Body1,
   Body2: Body2,
+  Display: Display,
   H1: H1,
   H2: H2,
   H3: H3,
```
Comment (Wendlereis) [inline src/shared/components/Typography/Typography.tsx:11]: **Question:** O noticed that other components with variants are lowercase. Does it make sense to set a standard for variant names?

Comment (frgiovanna) [reply src/shared/components/Typography/Typography.tsx:11]: That's fine to change this to lowercase too. But can I do it in another PR?

Comment (Wendlereis) [reply src/shared/components/Typography/Typography.tsx:11]: for sure

Comment (frgiovanna) [reply src/shared/components/Typography/Typography.tsx:11]: I added another one with this and other changes I was appalling to do: https://github.com/Wendlereis/minha-vez/pull/40

**Review by Wendlereis — APPROVED** [review]

**Pull request merged**